tests/draft4/refRemote.json

```diff
@@ -167,5 +167,25 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description":"$ref to a different schema version",
+        "schema":{
+            "type" : "object",
+            "properties" : {
+                "foo" : {"type" : "string"},
+                "bar" : {"$ref" : "http://localhost:1234/integer-draft6.json"}
+            }
+        },
+        "tests":[
+            {
+                "description": "schema 4 referencing schema 6",
+                "data" : {
+                    "foo" : "a value",
+                    "bar" : 15
```

Comment:
If we make this 10 with a validation outcome of `false` then I think we catch the inconsistency, similar to the other cases.

Reply:
I just added that locally and my validator still passed.  I'll update the PR with the new test case.